lib/components/ms-input/MsInput.vue

```diff
@@ -60,12 +61,14 @@ const props = defineProps<{
 const inputRef = ref();
 const errorMessage: Ref<Translatable> = ref('');
 const validity = ref(Validity.Intermediate);
-const lostFocus = ref(false);
+const lostFocus = ref(true);
 
 const emits = defineEmits<{
   (e: 'update:modelValue', value: string): void;
   (e: 'change', value: string): void;
   (e: 'onEnterKeyup', value: string): void;
+  (e: 'onFocusLost', value: boolean): void;
```

Comment:
You don't need booleans here.
If you have a signal like `focusChanged`, you can add a boolean `isFocused`.
Here, for `onFocusLost`, the boolean will always be true, and for `onFocus`, the boolean will always be false, which makes them useless.